source/reference/modules/engine/index.rst

```diff
@@ -1330,6 +1330,18 @@ Crafter Engine sports a built-in cache engine with an LRU (least recently used)
 
 .. note:: When running in Preview Mode (inside Studio for preview purposes), Crafter Engine's cache is disabled to help authors see their changes immediately.
 
+Engine has multiple caches:
+
+.. version_tag::
+    :label: Since
+    :version: 4.3.1
+
+- **user** cache for objects cached by Groovy code.
+  This cache is cleared manually by calling the :base_url:`cacheClear <_static/api/engine.html#tag/cache/operation/cacheClear>` API.
+  Active cache is only used on the user cache.
+- **system** cache for objects cached by Engine itself.
+  This cache is cleared when publishing occurs.
```

Comment:
System cache is cleared by the Deployer on publish calling the API, when the Deployer runs alongside Engine. When Engine runs in serverless mode and detects a change in the `deveployment-events.properties` file in S3, a new version of the system cache is built and the swapped with the old system cache version.